modules/apps/portal-workflow/portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/internal/runtime/integration/test/WorkflowDefinitionManagerTest.java

```diff
@@ -503,6 +530,29 @@ private void _assertValid(InputStream inputStream) throws Exception {
 			FileUtil.getBytes(inputStream));
 	}
 
+	private Closeable _disableScriptContentBeExecutedOrIncluded()
```

Comment:
Do you think it's worth putting this method in some util class and reusing it here and in the ObjectActionLocalServiceTest and ObjectValidationRuleLocalServiceTest classes? We are replicating this method in the test classes.

Reply:
I'll send a follow-up pull request with this change,

Thanks Camacho!

Reply:
No problem, I'll mark this as resolved and approve this PR then.